lib/Auth/Process/COmanageDbClient.php

```diff
@@ -851,7 +851,8 @@ private function couEntitlementAssemble($personRoles, &$state, $vo_name, $group_
         $state['Attributes']['eduPersonEntitlement'][] = $entitlement;
 
         if(!empty($personRoles['cou_id'])
-          && !empty($cou_tree_structure[ $personRoles['cou_id'] ])) {
+           && $role != 'vm_operator'
+           && !empty($cou_tree_structure[ $personRoles['cou_id'] ])) {
```

Comment:
Investigate alternative in order to allow applying the default VO roles that have been configured only at the top-level COU (i.e. VO)